cmd/dicomutil/main.go

```diff
@@ -17,7 +17,11 @@ import (
 	"github.com/suyashkumar/dicom/pkg/tag"
 )
 
+// Version is the current version of dicomutil, will be replaced in release step with current git commit hash or tag.
+var Version = "unknown"
```

Comment:
nit: it may be confusing to have both a `Version` and a `version` variable. Let's consider renaming this to something like `GitVersion` or `TaggedVersion` to help disambiguate? (Not capitalizing GIT because I don't think it's an initialism). 

```suggestion
var GitVersion = "unknown"
```

Reply:
renamed to `GitVersion` as suggested.